packages/material-ui/src/StepContent/StepContent.test.js

```diff
@@ -29,17 +29,17 @@ describe('<StepContent />', () => {
     const wrapper = shallow(
       <StepContent
         style={{ paddingRight: 200, color: 'purple', border: '1px solid tomato' }}
-        role="Tabpanel"
+        data-role="Tabpanel"
```

Comment:
Maybe the lint rule triggered because roles are case sensitive?

Reply:
Possibly. I believe the test is about forwarding the properties to the root element.